pkg/templates/generic_lookup_func.go

```diff
@@ -222,7 +222,7 @@ func (t *TemplateResolver) discoverAPIResources() ([]*metav1.APIResourceList, er
 		return nil, fmt.Errorf("failed to create the discovery client: %w", ddErr)
 	}
 
-	apiresourcelist, apiresourcelistErr := dd.ServerResources()
+	_, apiresourcelist, apiresourcelistErr := dd.ServerGroupsAndResources()
```

Comment:
This was required because `ServerResources` was removed but it just wrapped `ServerGroupsAndResources`.